src/universalinit/templates/express/config.yml

```
@@ -12,7 +12,7 @@ init_files: []
 init_minimal: "Minimal Express application initialized"
 
 run_tool:
-  command: "npm run dev"
+  command: "PORT=<port> HOST=0.0.0.0 npm run dev"
```

We're not missing the CI flag here?

```suggestion
  command: "CI=true PORT=<port> HOST=0.0.0.0 npm run dev"
```